examples/trimmer/dnsproxy/Makefile

```diff
@@ -37,7 +37,7 @@ ${OUT_DIR}:
 	mkdir -p ${OUT_DIR}
 
 test:
-	python3 test_dns.py
+	sudo python3 test_dns.py
```

Comment:
why sudo here?

Reply:
Running airtun without sudo gives the following error: unable to bind socket: Permission denied